api/lib/infrastructure/serializers/csv/csv-serializer.js

```diff
@@ -212,14 +212,22 @@ function _getComplementaryCertificationLabel(key, COMPLEMENTARY_CERTIFICATION_SU
   return key.replace(COMPLEMENTARY_CERTIFICATION_SUFFIX, '').trim();
 }
 
-function _verifyHeaders({ csvLineKeys, parsedCsvLine, headers }) {
+function _verifyHeaders({ csvLineKeys, parsedCsvLine, headers, hasBillingMode }) {
   csvLineKeys.forEach((key) => {
     if (parsedCsvLine[headers[key]] === undefined) {
+      if (__isBillingModeOptional(key, hasBillingMode)) {
+        return;
+      }
+
       throw new FileValidationError('CSV_HEADERS_NOT_VALID');
     }
   });
 }
 
+function __isBillingModeOptional(key, hasBillingMode) {
```

Comment:
```suggestion
function _isBillingModeOptional(key, hasBillingMode) {
```

Il y a un petit underscore en trop.

Reply:
C'est meme pas optional, on le veut pas du tout

Reply:
Si on le veut dans le contexte des centres SUP/PRO